msteams-platform/m365-apps/using-teams-client-sdk-preview.md

```diff
@@ -0,0 +1,354 @@
+| `settings.SaveEvent` interface | `pages.config.SaveEvent` (renamed)
+| `settings.RemoveEvent` interface | `pages.config.RemoveEvent` (renamed)
+| `settings.SaveParameters` interface | `pages.config.SaveParameters` (renamed)
+| `settings.SaveEventImpl` interface | `pages.config.SaveEventImpl` (renamed)
+
+| Original namespace `global (window)` | New namespace `pages.config` |
+| - | - |
+| `registerEnterSettingsHandler` | `pages.config.registerChangeConfigHandler` (renamed)
+
+#### *pages.backStack*
+
+| Original namespace `navigation` | New namespace `pages.backStack`  |
+| - | - |
+| `navigation.navigateBack` | `pages.backStack.navigateBack`
+
+| Original namespace `global (window)` | New namespace `pages.backStack`  |
+| - | - |
+| `registerBackButtonHandler` | `pages.backStack.registerBackButtonHandler`
+
+#### *pages.appButton*
+
+| Original namespace `global (window)` | New namespace `pages.appButton`  |
+| - | - |
+| `registerAppButtonClickHandler` | `pages.appButton.onClick` (renamed)
+| `registerAppButtonHoverEnterHandler` | `pages.appButton.onHoverEnter` (renamed)
+| `registerAppButtonHoverLeaveEnter` | `pages.appButton.onHoverLeave` (renamed)
+| `FrameContext` interface | `pages.appButton.FrameInfo` (renamed)) |
+
+### *dialog* namespace
+
+The TeamsJS *tasks* namespace has been renamed to *dialog*, and the following APIs have been renamed:
+
+| Original namespace `tasks` | New namespace `dialog`  |
+| - | - |
+| `tasks.startTask` | `dialog.open` (renamed) |
+| `tasks.submitTasks` | `dialog.submit` (renamed) |
+| `tasks.updateTasks` | `dialog.resize` (renamed) |
+| `tasks.TaskModuleDimension` enum | `dialog.DialogDimension` (renamed) |
+| `tasks.TaskInfo` interface | `dialog.DialogInfo` (renamed) |
+
+### *teamsCore* namespace
+
+To generalize the TeamsJS SDK to run other Microsoft 365 hosts such as Office and Outlook, Teams-specific functionality (originally in the *global* namespace) has been moved to a *teamsCore* namespace:
+
+| Original namespace `global (window)` | New namespace `teamsCore`  |
+| - | - |
+| `enablePrintCapability` | `teamsCore.enablePrintCapability`
+| `print` | `teamsCore.print`
+| `registerOnLoadHandler` | `teamsCore.registerOnLoadHandler`
+| `registerBeforeUnloadHandler` | `teamsCore.registerBeforeUnloadHandler`
+| `registerFocusEnterHandler` | `teamsCore.registerFocusEnterHandler`
+
+## Updates to the *Context* interface
+
+The `Context` interface has been moved to the `app` namespace and updated to group similar properties for better scalability as it runs in Outlook and Office, in addition to Teams.
+
+A new property `app.Context.app.host.name` has been added to enable personal tabs to differentiate user experience depending on the host application.
+
+You can also visualize the changes by reviewing the[`transformLegacyContextToAppContext`](https://github.com/OfficeDev/microsoft-teams-library-js/blob/2.0-preview/packages/teams-js/src/public/app.ts) function in the TeamsJS SDK Preview source.
```

Comment:
Need a space between "the" and "[`transformLegacyContextToAppContext`]"

Reply:
Thanks! fixed